info.yml

```diff
@@ -14,8 +14,8 @@ requires_shotgun_fields:
 
 
 # More verbose description of this item
-display_name: "ShotGrid Engine for ShotGrid Desktop v2"
-description: "ShotGrid Support in ShotGrid Desktop v2"
+display_name: "Flow Production Tracking Engine for Desktop app v2"
```

Comment:
Should this be "... Engine for Create app" now? @geffrak @bernie-laberge can you confirm this?

Reply:
`Engine for Flow Production Tracking Create` ?